### PR TITLE
Add Translation to Spanish

### PR DESCRIPTION
**Pull request by endlessloop2**
Add translation to Latin American (Neutral) Spanish. It is a mess, hope I don't mess something in the repo

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -8,78 +8,93 @@
...
 email: support@terracoin.io
 
 description:
   en: Terracoin is a secure, innovative, flexible, and easy cryptocurrency.
+  es: Terracoin es una criptomoneda segura, innovadora y flexible.
   fr: Terracoin est une crypto-monnaie sécurisée, innovante, flexible et facile.
   fi: Terracoin on turvallinen, innovatiivinen, joustava ja helppo kryptovaluutta.
 
 copyright:
+  en: Copyright &copy; 2018 Terracoin
```
Comment (TheSin-) [inline _config.yml]: I think you mean es here ?

File: es/disclaimer.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+layout: disclaimer
+ref: disclaimer
+order: 0
+bgclass: disclaimer
+
+# translate below, never translate before the first :
+
+title: Descargo de Responsabilidad
+lang: es
+permalink: /disclaimer/
```
Comment (TheSin-) [inline es/disclaimer.html]: it has to be unique, if disclaimer is disclaimer in es, then use /disclaimer-es/

File: es/exchanges.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+layout: exchanges
+ref: exchanges
+order: 3
+bgclass: exchanges
+
+# translate below, never translate before the first :
+
+title: Intercambios
+lang: es
+permalink: /exchanges/
```
Comment (TheSin-) [inline es/exchanges.html]: Same here has to be unique, if exchanges is exchanges in es use /exchanges-es/

File: es/inicio.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+layout: home
+ref: index
+order: 1
+bgclass: home
+
+# translate below, never translate before the first :
+
+title: Inicio
+lang: es
+permalink: /
```
Comment (TheSin-) [inline es/inicio.html]: this should be what ever index is in es, see the fr/acceuil.html as an example.

I assume /inicio/

File: es/trc-vitals.html [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+layout: vitals
+ref: trc-vitals
+order: 4
+bgclass: trc-vitals
+
+# translate below, never translate before the first :
+
+title: TRC Vitals #idk
+lang: es
+permalink: /trc-vitals/
```
Comment (TheSin-) [inline es/trc-vitals.html]: same as above, needs to be unique and reference es

File: es/trcpay.html [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+layout: trcpay
+ref: trcpay
+order: 0
+bgclass: trcpay
+
+# translate below, never translate before the first :
+
+title: TRCpay
+lang: es
+permalink: /trcpay/
```
Comment (TheSin-) [inline es/trcpay.html]: see other comments

**Review by TheSin- — COMMENTED** [review]
Please review the comments

**Comment by endlessloop2** [comment]
Do I have to change the config.yml only?

**Comment by TheSin-** [comment]
Can you not see all my notes on this page??  I'll screen shot it and send it to you on Mattermost

**Comment by TheSin-** [comment]
Thanks you Endless, accepted

**Pull request merged**